packages/itwinui-css/src/expandable-block/expandable-block.scss

```diff
@@ -0,0 +1,187 @@
+  cursor: pointer;
+  user-select: none;
+  color: var(--iui-color-text);
+  line-height: inherit;
+  font-family: inherit;
+  text-align: left;
+  background-color: var(--_iui-expandable-block-background-color);
+  border: var(--_iui-expandable-block-border);
+  border-radius: inherit;
+  border-bottom-left-radius: var(--_iui-expandable-block-header-bottom-border-radii, inherit);
+  border-bottom-right-radius: var(--_iui-expandable-block-header-bottom-border-radii, inherit);
+  transition: background-color var(--iui-duration-1) ease-out;
+  @media (forced-colors: active) {
+    --_iui-expandable-block-expander-icon-fill: CanvasText;
+  }
+
+  &:where(:hover) {
+    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-hover);
+    --_iui-expandable-block-header-caption-color: var(--iui-color-text-hover);
+    --_iui-expandable-block-header-label-color: var(--iui-color-text-hover);
+
+    background-color: var(--_iui-expandable-block-header-background-color-hover);
+
+    @media (forced-colors: active) {
+      --_iui-expandable-block-header-caption-color: Highlight;
+      --_iui-expandable-block-header-label-color: Highlight;
+      --_iui-expandable-block-expander-icon-fill: Highlight;
+      border-color: Highlight;
+    }
+  }
+
+  &[disabled],
+  &:disabled,
+  &[aria-disabled='true'] {
+    --_iui-expandable-block-expander-icon-fill: var(--iui-color-icon-disabled);
+    --_iui-expandable-block-header-caption-color: var(--iui-color-text-disabled);
+    --_iui-expandable-block-header-label-color: var(--iui-color-text-disabled);
+
+    cursor: not-allowed;
+    background-color: var(--_iui-expandable-block-header-background-color-disabled);
+    border-color: var(--iui-color-border-disabled);
+
+    @media (forced-colors: active) {
+      --_iui-expandable-block-expander-icon-fill: GrayText;
+      --_iui-expandable-block-header-caption-color: GrayText;
+      --_iui-expandable-block-header-label-color: GrayText;
+      border-color: GrayText;
+    }
+  }
+}
+
+/// @arg $type Can be one of: expander, status
+@mixin iui-expandable-block-header-icons($type) {
+  @include iui-icon-style('m');
+
+  @if $type == 'expander' {
+    width: var(--_iui-expandable-block-expander-icon-size);
+    height: var(--_iui-expandable-block-expander-icon-size);
+    fill: var(--_iui-expandable-block-expander-icon-fill);
+    transform: var(--_iui-expandable-block-expander-icon-transform, none);
+    transition: fill var(--iui-duration-1) ease-out;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: fill var(--iui-duration-1) ease-out, transform var(--iui-duration-1) ease-out;
+    }
+  }
+
+  @if $type == 'status' {
+    @media (forced-colors: active) {
+      fill: CanvasText;
+    }
+  }
+}
+
+@mixin iui-expandable-block-header-label {
+  display: flex;
+  flex-direction: column;
+  flex: auto;
+  min-width: 0;
+  font-size: var(--iui-font-size-2);
+  color: var(--_iui-expandable-block-header-label-color, inherit);
```

Comment:
`--_iui-expandable-block-header-label-color` not found. So forced colors don't get applied

![image](https://user-images.githubusercontent.com/45748283/203425816-415118ba-c063-4a1d-8e61-ef854e923dfa.png)